accelerator/apis.h

```diff
@@ -28,14 +28,16 @@ extern "C" {
  * Generate and return an unused address from the seed. An unused address means
  * the address does not have any transaction with it yet.
  *
+ * @param[in] weight IOTA API parameter configurations
```

Comment:
Is `weight` a comprehensive name for passing to TA APIs?

Reply:
I was thinking `account` or `network`. Not sure if it's better choice.

Reply:
`weight` is not key factor influencing Tangle network. We need to apply general terminologies though.

Reply:
`network` then. Sorry, I have hard time came up with good name for abstraction. :sob: